lib/services/android/gradle-build-args-service.ts

```diff
@@ -95,7 +104,10 @@ export class GradleBuildArgsService implements IGradleBuildArgsService {
 	}
 
 	private getBuildTaskName(buildData: IAndroidBuildData): string {
-		const baseTaskName = buildData.androidBundle ? "bundle" : "assemble";
+		let baseTaskName = buildData.androidBundle ? "bundle" : "assemble";
+		if (buildData.gradleFlavor) {
+			baseTaskName += buildData.gradleFlavor[0].toUpperCase() + buildData.gradleFlavor.slice(1);
```

Comment:
Why do we need to uppercase the first letter here? Shouldn't it just be passed in the correct casing?

Reply:
we are adding a flavor so it should become `bundleMyflavorRelease` when flavor is `myflavor`